dev/conductor/core/lib/src/codesign.dart

```diff
@@ -75,6 +75,10 @@ class CodesignCommand extends Command<void> {
   FrameworkRepository get framework {
     return _framework ??= FrameworkRepository(
       checkouts,
+      upstreamRemote: Remote(
```

Comment:
because this argument was not being passed, it was always defaulting to cloning from https://github.com/flutter/flutter. This is a problem if a PR hasn't merged yet, because it means the commit we want to test doesn't exist on flutter/flutter.